### ISSUE TITLE
Webpack plugin

### ISSUE THREAD

**Issue by Agamennon** [kodyl/stilr#11]
Thanks for this, might i suggest a webpack plugin,  so that not only i don't need to:
    ``` if (process.env !== 'production') {
       const prefixedCSS = postcss(autoprefixer()).process( StyleSheet.render() ).css;
       stylesheet.textContent = prefixedCSS;
     } ```
but also to ether on development or  production, to leverage the webpack build process

**Comment by chriskjaer** [comment]
True, a webpack plugin would be awesome for a lot of workflows I guess.

Unfortunately I'm not really that familiar with writing webpack plugins, so if you're up for the task of writing one I will be trilled.

**Comment by seethroughtrees** [comment]
This is a really clever approach.  Nice job...  

This would definitely be best utilized in a webpack plugin.  It would also definitely help with adoption. Hopefully someone with a decent amount of experience in that area can help out here.

**Comment by chcokr** [comment]
Are you guys just wanting the body of the `if` block to be added via a webpack plugin? I'm asking because I can help, but I want to point out that everyone defines "production" differently: some use `process.env`, some use `process.env.NODE_ENV`, etc.

**Comment by msikma** [comment]
It should be `process.env.NODE_ENV`, most likely. It's always [an object](https://nodejs.org/api/process.html#process_process_env).
